docs/x509/verification.rst

```diff
@@ -111,12 +111,22 @@ the root of trust:
 
     .. versionadded:: 43.0.0
 
-    .. attribute:: subjects
+    .. versionchanged:: 44.0.0
+        Renamed `subjects` to :attr:`sans`. 
+        Made `sans` optional, added :attr:`subject`.
 
-        :type: list of :class:`~cryptography.x509.GeneralName`
+    .. attribute:: subject
+
+        :type: :class:`~cryptography.x509.Name`
+
+        The subject presented in the verified client's certificate.
+
+    .. attribute:: sans
+
+        :type: list of :class:`~cryptography.x509.GeneralName` or None
```

Comment:
Yeah, agreed. This is the thing that I pointed out in [my previous comment ](https://github.com/pyca/cryptography/pull/11559#discussion_r1753626903). 

I think having just the `subjects` field but making it optional is fine. There could potentially be two `VerifiedClient` classes, one used with `PolicyBuilder`, and the other used with `CustomPolicyBuilder`, since `subjects` only needs to be optional in the first case, but personally I think the increased implementation complexity is just not worth it.

Reply:
I have removed `subject` and renamed `sans` back to `subjects` (but kept it optional). I'm leaving the conversation unresolved in case there are some other concerns related to this.